tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.py

```diff
@@ -0,0 +1,76 @@
+# cat test_realtime_unsupported.py
```

Comment:
```suggestion
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/a46ae8a444c0c0a7f68436a4ccee7af9780437c6)